src/app/modules/forms/ix-forms/services/form-error-handler.service.ts

```diff
@@ -146,9 +148,18 @@ export class FormErrorHandlerService {
     });
     control.markAsTouched();
 
-    const element = this.formService.getElementByControlName(field);
+    // Notify EditableComponents that might contain this field
+    this.notifyEditablesOfValidationError(field);
+
+    // Try to get element from IxFormService first, then fallback to querySelector
+    let element = this.formService.getElementByControlName(field);
+    if (!element && this.document?.querySelector) {
+      // Fallback: try to find element by formControlName attribute
+      element = this.document.querySelector(`[formControlName="${field}"]`) as HTMLElement;
+    }
+
     if (!element) {
-      console.warn(`getElementByControlName Could not find element for ${field}.`);
+      console.warn(`Could not find DOM element for field ${field}.`);
       this.needToShowError = true;
```

Comment:
Perhaps if we cant find we can display at t he top of the form instead of a error dialog?

Reply:
@william-gr - for all other forms we are displaying the dialogs on some errors (without field mentioning) - should we replace it across all forms? (to show the error inline at the top of the form or near the submit button)

Reply:
@william-gr  ah, in this case I'll try to fix it here, if API mentions the field. Seems like there are some issues with conditions..